Sources/Plasma/PubUtilLib/plResMgr/plLocalization.cpp

```diff
@@ -45,41 +45,51 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "HeadSpin.h"
 #include "plFileSystem.h"
 
+#include <algorithm>
 #include <string_theory/string>
 #include <string_theory/string_stream>
 
 plLocalization::Language plLocalization::fLanguage = plLocalization::kEnglish;
 
-const ST::string plLocalization::fLangTags[] =
-{
+const std::array<ST::string, plLocalization::kNumLanguages> plLocalization::fLangTags = {
     ST_LITERAL("_eng"), // kEnglish
     ST_LITERAL("_fre"), // kFrench
     ST_LITERAL("_ger"), // kGerman
     ST_LITERAL("_spa"), // kSpanish
     ST_LITERAL("_ita"), // kItalian
-    ST_LITERAL("_jpn")  // kJapanese
+    ST_LITERAL("_jpn"), // kJapanese
+    ST_LITERAL("_dch"), // kDutch
+    ST_LITERAL("_rus"), // kRussian
+    ST_LITERAL("_pol"), // kPolish
+    ST_LITERAL("_czk"), // kCzech
 };
 const int kLangTagLen = 4;
 
 // ISO 639, e.g. used in video tracks
-std::set<ST::string> plLocalization::fLangCodes[] =
-{
-    {"eng", "en"},
-    {"fre", "fra", "fr"},
-    {"ger", "deu", "de"},
-    {"spa", "es"},
-    {"ita", "it"},
-    {"jpn", "ja"}
-};
-
-const ST::string plLocalization::fLangNames[] =
-{
-    ST_LITERAL("English"), // kEnglish
-    ST_LITERAL("French"),  // kFrench
-    ST_LITERAL("German"),  // kGerman
-    ST_LITERAL("Spanish"), // kSpanish
-    ST_LITERAL("Italian"), // kItalian
-    ST_LITERAL("Japanese") // kJapanese
+const std::array<std::set<ST::string>, plLocalization::kNumLanguages> plLocalization::fLangCodes = {{
```

Comment:
Why not an array of `std::unordered_set`?

Reply:
It was already an ordered `std::set` before, so I kept it that way. Looks like nothing relies on the order though, so it should be safe to change to `std::unordered_set`.